app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -281,12 +281,18 @@ ns.Collection = do (ko
       @cwic(@_isCwic(jsonObj))
 
       @nrt = jsonObj.collection_data_type == "NEAR_REAL_TIME"
-      @granuleCount(jsonObj.granule_count)
 
       for own key, value of jsonObj
         this[key] = value unless ko.isObservable(this[key])
 
+      if @_isCwic(jsonObj)
```

Comment:
CMR doesn't report ```granule_counts``` and ```has_granules``` if it doesn't hold any. So I hacked around this to provide ```granule_counts(1)``` so that the collection behaves like other CMR collections with granules.